homeassistant/components/ecodevices/translations/fr.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Please remove this translation, initial PR's should only have `strings.json` and `en.json` for testing. Other languages will be added later via Lokalize :).